### DISCUSSION TITLE
Enabling Misskey integration (despite I'm the one who made it)

### DISCUSSION THREAD

**Discussion by reinhart1010** [upptime/upptime#678] [Q&A]
Hi! I've previously contributed upptime/uptime-monitor#193 and upptime/uptime-monitor#195 before, and the worst part is I can't even get my self-made Misskey notifier to work.

<img width="779" alt="image" src="https://user-images.githubusercontent.com/17312341/202913388-a880986d-d033-4f87-abb2-b0e310f1d812.png">

I've already set `NOTIFICATION_MISSKEY` and `NOTIFICATION_MISSKEY_NOTE` to `true`, then re-checked how my code was working. But since the [original repo](https://github.com/reinhart1010/binusmayadown/) was created before I pushed these PRs, are there any missing steps which I haven't done yet?

**Comment by AnandChowdhary** [comment]
You're right, we need to add these environment variables to the workflows in the .github directory.
